server/development.ini

```diff
@@ -13,9 +13,15 @@ pyramid.debug_routematch = false
 pyramid.default_locale_name = en
 pyramid.includes =
     pyramid_debugtoolbar
-
+    
 sqlalchemy.url = postgresql://admin:1@eventme_postgres/eventme
 
+mail.host = smtp.gmail.com
+mail.username = eventmerv@gmail.com
+mail.password = Docker666Vim
```

Comment:
It'a always a bad idea to store passwords in public repositories, where everyone can see them.

The idea is next:

1. Rename development.ini to development.ini.sample
2. Add development.ini to .gitignore
3. Add instruction to README to copy development.ini.sample to development.ini and fill credentails before first deployment
4. ....
5. Profit

Reply:
Can I just make empty fields?

Reply:
No, it's not enough. As every developer after updating those fields will have this file marked as changed by git and can accidentally commit it.